### PR TITLE
Updating focalboard icon and text

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
Updating focalboard icon and text

#### Ticket Link
https://github.com/mattermost/focalboard/issues/909
https://github.com/mattermost/focalboard/issues/900

#### Screenshot
Screenshot 2021-08-10 at 8 09 25 PM

### CODE REVIEW

File: webapp/src/styles/_compass-icons.scss [hunk 1]
```diff
@@ -17,44 +17,43 @@
 @media screen and (-webkit-min-device-pixel-ratio:0) {
   @font-face {
     font-family: 'compass-icons';
-    src: url('../fonts/compass-icons/compass-icons.svg?45182295#compass-icons') format('svg');
+    src: url('../font/compass-icons.svg?94622077#compass-icons') format('svg');
```
Comment (jespino) [inline webapp/src/styles/_compass-icons.scss:20]: This change looks like not needed, it should keep untouched like the other lines at the beginning of the file.

**Review by jespino — CHANGES_REQUESTED** [review]
A tiny change needed, other than that, looks good to me.

File: webapp/src/styles/_compass-icons.scss (hunk 1 above)
Comment (asaadmahmood) [reply webapp/src/styles/_compass-icons.scss]: @jespino Done.

**Review by jespino — APPROVED** [review]
LGTM

**Pull request merged**